relay-server/server/relayServer.go

```diff
@@ -619,6 +624,10 @@ func (rs *RelayServer) ServeLogFeeds() {
 // =============== //
 
 func connectToKubeArmor(nodeIP, port string) error {
+
+	ClientListLock.Lock()
```

Comment:
we dont need a mutex lock here 
this go routine is not trying to access any global varibale 
also watchlogs has an infinite loop